### DISCUSSION TITLE
An uninstall script

### DISCUSSION THREAD

**Discussion by wonderlandKid** [NvChad/NvChad#1740] [Q&A] [answered]
Sorry for my poor english. 

The plugins always change so I really dont know where the plugins are. And I'm new to vim. So I hope Nvchad can have a simple script to uninstall. It seems simple to do but diffcult for me.

Anyway I read many README and solve this problem but I think doing this can be help for those new vimers using Nvchad like me. 😃 We can reinstall easily.

**Comment by prsh11** [comment, marked as answer]
```bash
# linux/macos (unix)
rm -rf ~/.config/nvim
rm -rf ~/.local/share/nvim
rm -rf ~/.cache/nvim

# windows
rd -r ~\AppData\Local\nvim
rd -r ~\AppData\Local\nvim-data
```
run these commands for your operating system.
